Marlin/src/HAL/STM32F1/timers.h

```diff
@@ -40,6 +40,7 @@
  */
 
 typedef uint16_t hal_timer_t;
+#define TIMER_NUM          14 // Maple supports up to 14 timers
```

Comment:
14 for STM32_XL_DENSITY (F/G chips), 8 for HIGH density (C D E)

Reply:
We could do an enum but this PR is just for Maple life support of existing systems. They should eventually switch to the more supportable HAL_STM32 lib in the long term. I will update to text with the more descriptive text.

Reply:
or just TERN(STM32_XL_DENSITY, 14, 8)

Reply:
Done.

Reply:
you can also cancel STM32/fast_pwm.cpp changes which are already in the other PR, will conflict again :p

Reply:
Not sure on that git process, git Noobish here ..

Reply:
yep, nvm... this mess on these PRs lol

Reply:
Do you mean to pull in file origin/bugfix-2.0.x Marlin/src/HAL/STM32/fast_pwm.cpp to this PR and commit ament?

Reply:
yes, current marlin file to avoid another change in this PR... (but keep the .h which has a new change)

Reply:
Maybe let Scott adjust that .. lol

Reply:
yes... else you can just save it from github link under.. and commit...

Reply:
Better now.

Reply:
as reference... because finding things in this repo is always a mess... https://github.com/rogerclarkmelbourne/Arduino_STM32/blob/master/STM32F1/system/libmaple/stm32f1/include/series/timer.h#L119